README.md

```diff
@@ -201,7 +206,7 @@ $ chisel server --help
 
 
 <!--tmpl,code=plain:echo "$ chisel client --help" && go run main.go client --help | sed 's#0.0.0-src (go1\..*)#X.Y.Z#' -->
-``` plain 
+``` plain
```

Comment:
these spacing edits come from your IDE i think,
can we revert them to reduce the size of the change set

easiest way is to execute the markdown template https://github.com/jpillora/md-tmpl